docs/src/main/asciidoc/building-native-image.adoc

```diff
@@ -676,28 +676,28 @@ docker run \
   --entrypoint bin/sh \
   quay.io/quarkus/ubi-quarkus-native-image:{graalvm-flavor} \ <3>
   -c "native-image $(cat native-image.args) -J-Xmx4g" <4>
----
+----
 
 <1> Mount the host's directory `target/native-image` to the container's `/work`. Thus, the generated binary will also be written to this directory.
 <2> Switch the working directory to `/work`, which we have mounted in <1>.
 <3> Use the `quay.io/quarkus/ubi-quarkus-native-image:{graalvm-flavor}` docker image introduced in <<#multistage-docker,Using a multi-stage Docker build>> to build the native image.
-<4> Call `native-image` with the content of file `native-image.args` as arguments. We also supply an additional argument to limit the process's maximum memory limitation to 4 Gigabyte.
+<4> Call `native-image` with the content of file `native-image.args` as arguments. We also supply an additional argument to limit the process's maximum memory limitation of 4 Gigabyte (this may vary depending on the project being built and the machine building it).
 
-In a CI/CD setup, we would:
+[WARNING]
+====
+If you are running on a Windows machine, please keep in mind that the binary was created within a Linux docker container.
+Hence, the binary will not be executable on the host Windows machine.
+====
 
-1. register the output of the step executing `./mvnw ...` command (i.e. directory `target/native-image`) as build artifact,
-2. require this artifact in the step executing the `native-image ...` command, and
-3. register the output of the step executing the `native-image ...` command (i.e. files matching `target/*runner`) as build artifact.
+A CI/CD pipeline a high level overview of the various steps would look like so:
```

Comment:
I think there is a word missing after `pipeline`?

Reply:
Good catch, fixed

Reply:
I don't see it fixed?

Reply:
I guess I hadn't pushed 🤦🏼 . Should be OK now